### ISSUE TITLE
Training is not working. Loss gets worse

### ISSUE THREAD

**Issue by kalle264** [N0vel/weighted-hausdorff-distance-tensorflow-keras-loss#2]
Hi,
did you use your implementation successfully for training?

It's working for me when I just calculate the loss for a prediction of a pretrained model, but when I want to train with it, the loss increases from batch to batch and then just stays at the diagonal distance for every following epoch.

My code works just fine with a different loss.
Do you have any idea what could cause that problem?

**Comment by N0vel** [comment]
Hi,
Try to tune learning rate and the parameters of the loss itself (it's a weighted sum of two components)
